embedded-kafka/src/test/java/com/playtika/test/kafka/EmbeddedKafkaTests.java

```diff
@@ -23,190 +23,59 @@
-        this.kafkaTopicsConfigurer.createTopics(Collections.singletonList(topicToCreate));
-
-        assertThatTopicExists(topicToCreate);
-    }
+    private ZookeeperConfigurationProperties zookeeperProperties;
 
-    @Test
-    public void should_sendAndConsumeMessage() throws Exception {
-        sendMessage(TOPIC, MESSAGE);
-
-        String consumedMessage = consumeMessage(TOPIC);
-
-        assertThat(consumedMessage).isEqualTo(MESSAGE);
-    }
+    @Autowired
+    private KafkaConfigurationProperties kafkaProperties;
 
     @Test
-    public void shouldEmulateLatencyOnSend() throws Exception {
-        kafkaNetworkTestOperations.withNetworkLatency(ofMillis(1000),
-                () -> assertThat(durationOf(() -> sendMessage(TOPIC, "abc0")))
-                        .isGreaterThan(1000L)
-        );
-
-        assertThat(durationOf(() -> sendMessage(TOPIC, "abc1")))
-                .isLessThan(200L);
-
-        assertThat(consumeMessages(TOPIC)).containsExactly("abc0", "abc1");
-    }
-
-    private void assertThatTopicExists(String topicName) throws Exception {
-        ListTopicsResult result = adminClient.listTopics();
-        Set<String> topics = result.names().get(10, TimeUnit.SECONDS);
-        assertThat(topics).contains(topicName);
-    }
-
-    private void sendMessage(String topic, String message) throws Exception {
-        try (KafkaProducer<String, String> kafkaProducer = createProducer()) {
-
-            kafkaProducer.send(new ProducerRecord<>(topic, message)).get();
-        }
-    }
-
-    private String consumeMessage(String topic) {
-        return consumeMessages(topic).stream()
-                .findFirst()
-                .orElseThrow(() -> new IllegalStateException("no message received"));
-    }
-
-    private List<String> consumeMessages(String topic) {
-        try (KafkaConsumer<String, String> consumer = createConsumer(topic)) {
-            return pollForRecords(consumer).stream()
-                    .map(ConsumerRecord::value)
-                    .collect(Collectors.toList());
-        }
-    }
-
-    private KafkaProducer<String, String> createProducer() {
-        Map<String, Object> producerConfiguration = getKafkaProducerConfiguration();
-        return new KafkaProducer<>(producerConfiguration);
-    }
-
-    private KafkaConsumer<String, String> createConsumer(String topic) {
-        Map<String, Object> consumerConfiguration = getKafkaConsumerConfiguration();
-        Properties properties = new Properties();
-        properties.putAll(consumerConfiguration);
-        KafkaConsumer<String, String> consumer = new KafkaConsumer<>(properties);
-        consumer.subscribe(singleton(topic));
-        return consumer;
-    }
-
-    private static <K, V> List<ConsumerRecord<K, V>> pollForRecords(KafkaConsumer<K, V> consumer) {
-        ConsumerRecords<K, V> received = consumer.poll(Duration.ofSeconds(10));
-        return received == null ? emptyList() : Lists.newArrayList(received);
+    @Order(5)
```

Comment:
need to avoid ordering in tests

Reply:
Ok. A single constrain I have, tests shouldBindToFileSystem/shouldNotBindToFileSystem must be executed after all others.
I will try to achieve it using @AfterAll. Or maybe you have other suggestions ?

Reply:
Done. Removed order from tests